### PR TITLE
Adding task to debug a Stateful Set.

### PR DESCRIPTION
**Pull request by foxish**
Adding task to debug a Stateful Set.

/cc @kow3ns @enisoc @kubernetes/sig-apps

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/1811)

### CODE REVIEW

File: docs/tasks/stateful-sets/debugging-a-statefulset.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
+---
```
Comment (enisoc) [inline docs/tasks/stateful-sets/debugging-a-statefulset.md]: Are you going to add it to Nav in this PR?

File: docs/tasks/stateful-sets/debugging-a-statefulset.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+- erictune
+- foxish
+- janetkuo
+- kow3ns
+- smarterclayton
+
+---
+
+{% capture overview %}
+This task shows you how to debug a Stateful Set.
```
Comment (enisoc) [inline docs/tasks/stateful-sets/debugging-a-statefulset.md]: Don't forget to update to CamelCase as per https://github.com/kubernetes/kubernetes.github.io/pull/1807.

File: docs/tasks/stateful-sets/debugging-a-statefulset.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+{% endcapture %}
+
+{% capture prerequisites %}
+
+* If you are not using Stateful Sets, you can skip this task now.
```
Comment (enisoc) [inline docs/tasks/stateful-sets/debugging-a-statefulset.md]: Maybe word this as: You should already have a StatefulSet running that you want to investigate. You could link to the StatefulSet concept page too.

File: docs/tasks/stateful-sets/debugging-a-statefulset.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+{% capture steps %}
+
+### Debugging a Stateful Set
+
+In order to list all the pods which belong to a Stateful Set, which have a label `app=myapp` set on them, you can use the following: 
+
+```shell
+kubectl get pods -l app=myapp
+```
+If you find that any Pods listed are in `Unknown` or `Terminating` state for an extended period of time, refer to the [Deleting Stateful Set Pods](/docs/tasks/stateful-sets/deleting-pods/) task for instructions on how to deal with them. You can debug individual Pods in a Stateful Set using the [debugging Pods guide](docs/user-guide/debugging-pods-and-replication-controllers/#debugging-pods).
```
Comment (enisoc) [inline docs/tasks/stateful-sets/debugging-a-statefulset.md]: The two links to other pages are inconsistent in capitalization and inclusion of "task" or "guide" in the linkified span.

File: docs/tasks/stateful-sets/debugging-a-statefulset.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+        app: my-app
+      annotations:
+        pod.alpha.kubernetes.io/initialized: "false"
+...
+...
+... 
+
+```
+
+After setting the annotation, if you create the Stateful Set, you can wait for each Pod to come up and verify that it has initialized correctly. The Stateful Set will not create any subsequent Pods till the the debug annotation is set to `"true"` (or removed) on each Pod that has already been created.
```
Comment (enisoc) [inline docs/tasks/stateful-sets/debugging-a-statefulset.md]: Double "the".

File: docs/tasks/stateful-sets/debugging-a-statefulset.md (hunk 1 above)
Comment (foxish) [inline docs/tasks/stateful-sets/debugging-a-statefulset.md]: I'll add to index and tasks at the very end, right before merging, so as to avoid rebases.

**Comment by foxish** [comment]
@enisoc Addressed comments. Thanks! 
@devin-donnelly PTAL

**Comment by foxish** [comment]
Preview: https://deploy-preview-1811--kubernetes-io-vnext-staging.netlify.com/docs/tasks/manage-stateful-set/debugging-a-statefulset/

File: docs/tasks/stateful-sets/debugging-a-statefulset.md (hunk 1 above)
Comment (foxish) [inline docs/tasks/stateful-sets/debugging-a-statefulset.md]: On second thought, added in the index & tasks page changes.

**Comment by devin-donnelly** [comment]
Looking good. Does @enisoc want to give this a tech LGTM before I merge?

**Comment by enisoc** [comment]
LGTM

**Comment by foxish** [comment]
@devin-donnelly ok to merge?

**Comment by foxish** [comment]
@steveperry-53 Done. Thanks!

**Comment by erictune** [comment]
Nice.  LGTM.

**Comment by devin-donnelly** [comment]
Okay to merge once merge conflicts are fixed.

**Pull request merged**